cfgov/preprocessed/css/molecules/form-field-with-button.less

```diff
@@ -0,0 +1,60 @@
+/* topdoc
+  name: Form Field with Button
+  family: cfgov-molecules
+  patterns:
+    - name: Default example
+      markup: |
+		<div class="m-form-field-with-button">
+		    <div class="form-group">
+		        <label for="name">
+		            <b></b>(required)
+		        </label>
+		        <input id="" type="" name="" placeholder=""
+		        class="m-form-field-with-button_field">
+		    </div>
+			<p class="short-desc">
+			</p>
+		    <input class="btn btn__full" type="" value="">
+		</div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .m-form-field-with-button
+            .form-group
+              label
+              	b
+              .m-form-field-with-button_field
+            .short-desc
+            .btn btn__full
+  tags:
+    - cfgov-molecules
+*/
+
+@margin__em: unit( @grid_gutter-width / @base-font-size-px, em );
+
+.m-form-field-with-button {
+    input[type="text"],
+    input[type="search"],
+    input[type="email"],
+    input[type="url"],
+    input[type="tel"],
+    input[type="number"] {
+        width: calc(100% - 10px);
```

Comment:
When set to 100% it doesn't respect the padding and results in:

<img width="411" alt="screen shot 2015-10-15 at 9 25 53 am" src="https://cloud.githubusercontent.com/assets/1696212/10514925/dc6a8baa-731e-11e5-89b7-491629560e20.png">